src/Simplex_tree/include/gudhi/Simplex_tree.h

```diff
@@ -681,6 +692,14 @@ class Simplex_tree {
     return dim - 1;
```

Comment:
I wonder why we don't initialize `dim` to -1 instead of 0 to avoid this final subtraction, but it isn't new in this PR.

Reply:
I don't know either. I was asking myself the same question when I documented the method. It is probably just easier to visualize what happens that way.